src/server/req_rescq.c

```diff
@@ -242,17 +242,20 @@ cnvrt_timer_init()
  * @retval	non-zero	: error code if problem occurs
  */
 int
-assign_resv_resc(resc_resv *presv, char *vnodes)
+assign_resv_resc(resc_resv *presv, char *vnodes, int svr_init)
```

Comment:
we should update the doxygen to accommodate svr_init

Reply:
Done